ring/mk/llvm_check.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/bash  
+
+
+function llvm_version_check() {
```

Comment:
We require the version >= 10. Can we enforce that? Also, do you think we can also check the clang version?

Reply:
How about upgrading the version if the system is less then 10?

Reply:
I think it's fine to prompt user they have to upgrade. Automatic upgrade is a bit too aggressive I think

Reply:
@h4x3rotab implemented a version  check in latest